src/services/folderservices.ts

```diff
@@ -49,6 +49,7 @@ const createFolder = async ({
     });
   return folder;
 };
+
```

Comment:
about this what if the person wants to just update the name without updating the description or even vice versa?